snaps/tutorials/transaction-insights.md

```diff
@@ -12,22 +12,20 @@ The Snap provides transaction insights in the MetaMask transaction window.
 ## Prerequisites
 
 - [MetaMask Flask installed](../get-started/install-flask.md)
-
 - An account on your MetaMask Flask instance with testnet ETH
-
   :::tip
   You can use [Infura's Sepolia faucet](https://www.infura.io/faucet) to get Sepolia ETH.
   :::
-
 - A text editor (for example, [VS Code](https://code.visualstudio.com/))
-
+- [Node](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 18.16 or later
 - [Yarn](https://yarnpkg.com/) version 3
 
 ## Steps
 
 ### 1. Set up the project
 
-Create a new Snap project using the [`@metamask/create-snap` CLI](https://github.com/MetaMask/snaps/tree/main/packages/create-snap) by running:
+Create a new Snap project using the [`@metamask/create-snap`](https://github.com/MetaMask/snaps/tree/main/packages/create-snap)
+CLI by running:
```

Comment:
Same comment as before, this is the yarn or npm CLI here.

Reply:
Addressed.